components/channel_notifications_modal/components/describe.jsx

```diff
@@ -87,4 +95,5 @@ Describe.propTypes = {
     ignoreChannelMentions: PropTypes.string,
     memberNotifyLevel: PropTypes.string.isRequired,
     section: PropTypes.string.isRequired,
+    isCollapsed: PropTypes.bool,
```

Comment:
Changing this to a boolean as i don't anticipate a need to pass other values for view other than collapsed.